hotsos/client.py

```diff
@@ -284,6 +284,13 @@ def get(self, fmt='yaml', html_escape=False, minimal_mode=None,
 
         return filtered
 
+    def temp_log_filename(self):
```

Comment:
nit: i feel like this should live in hotsos.core.log but can do as a separate patch